anago

```diff
@@ -364,10 +364,10 @@ prepare_tree () {
   # the NON-master branch itself, versionize the docs
   if [[ "$PARENT_BRANCH" == master && "$current_branch" != "master" ]]; then
     logecho -n "Versionizing docs for ${RELEASE_VERSION[$label]}: "
-    logrun -s $TREE_ROOT/build/versionize-docs.sh ${RELEASE_VERSION[$label]}
+    logrun -s $TREE_ROOT/build/versionize-docs.sh $current_branch
```

Comment:
It is in code that already uses that style.  Thankfully in this repo I choose not to use that silliness.